package.json

```diff
@@ -48,19 +51,37 @@
     "@commitlint/config-conventional": "^17.0.2",
     "@evilmartians/lefthook": "^1.2.2",
     "@react-native-community/eslint-config": "^3.0.2",
+    "@storybook/addon-actions": "^6.5.14",
+    "@storybook/addon-controls": "^6.5.14",
+    "@storybook/addon-essentials": "^6.5.14",
+    "@storybook/addon-links": "^6.5.14",
+    "@storybook/addon-ondevice-actions": "next",
+    "@storybook/addon-ondevice-backgrounds": "next",
+    "@storybook/addon-ondevice-controls": "next",
+    "@storybook/addon-ondevice-notes": "next",
+    "@storybook/addon-react-native-web": "^0.0.19",
+    "@storybook/react-native": "next",
+    "@storybook/react": "^6.5.14",
+    "@storybook/testing-library": "^0.0.13",
```

Comment:
Why use too many addons
Will using `next` cause any issue when those version goes higher than the rest?

Reply:
the Storybook I'm using `V6.5`. 
but lib `@storybook/react-native` support for version `6.5` still be Beta-Version so i need using `next`